### PR TITLE
Fix tests for screenshots (EXPOSUREAPP-11295)

### PR DESCRIPTION
**Pull request by schauersbergern**
Fix tests. This PR can be reviewed and merged

### CODE REVIEW

**Review by mtwalli — DISMISSED** [review]

**Comment by SamuraiKek** [comment]
LGTM so far, but have a look at the latest comments in the ticket? Looks like some screen is not scrolled all the way down. Will approve once that's clarified.

**Review by BMItr — DISMISSED** [review]

**Review by SamuraiKek — DISMISSED** [review]

File: .circleci/config.yml
Comment (mtwalli) [inline .circleci/config.yml:495]: should be removed

Comment (schauersbergern) [reply .circleci/config.yml]: It's removed now. It just was present because I had to regenerate the screenshots. For that cause I also added the "do not merge" label again. But now it can be merged

**Review by mtwalli — APPROVED** [review]

**Pull request merged**
